src/app/components/Providers/SentryProvider.jsx

```diff
@@ -15,7 +15,7 @@ function CrashPage() {
   );
 }
 
-export default class SentryBoundary extends React.Component {
+class SentryProvider extends React.Component {
```

Comment:
This is indeed a `Boundary`, not a `Provider`

Reply:
I'm well aware. But to keep consistent naming it makes sense.
Similarly to how you added the 'GoogleAnalyticsProvider' which 'provides' GA without specifically being a Context Provider. It 'provides' an error boundary.

Reply:
Reverted.